### ISSUE TITLE
Path to `--buildConfig` is not output in log although code indicates it should

### ISSUE THREAD

**Issue by janpio** [apache/cordova-ios#648]
Just noticed when looking through some CI logs:

```
$ cordova compile ios --device --buildConfig=build.json --buildFlag='-UseModernBuildSystem=0' --verbose
No scripts found for hook "before_compile".
PlatformApi successfully found for platform ios
Reading build config file:
Building project: /tmp/app/platforms/ios/HelloCordova.xcworkspace
...
```

Code for the output that should include the filename: https://github.com/apache/cordova-ios/blob/da5b69d62f2732039e9c82e603a24e291b086847/bin/templates/scripts/cordova/lib/build.js#L125

Note: This is Cordova CLI 8.1.1 and Cordova iOS 5.0.1

**Comment by erisu** [comment]
One thing I noticed is it is 3 arguments.

```events.emit('log', 'Reading build config file:', path.resolve(buildOpts.buildConfig));```
- log
- Reading build config file:
- path.resolve(buildOpts.buildConfig)

In almost every use case where I use events.emit, its only 2 arguments. Never knew if it could take more then 2..
This might work:

```events.emit('log', `Reading build config file: ${path.resolve(buildOpts.buildConfig)}`);```

I noticed that the OSX platform repo might capture the emit and pipe it to `console.log` so in theory that would accept N number of args but I don't know what default does.
